src/Fable.Transforms/FSharp2Fable.fs

```diff
@@ -1438,14 +1438,13 @@ let private addUsedRootName (com: Compiler) name (usedRootNames: Set<string>) =
     Set.add name usedRootNames
 
 // Entities that are not output to other languages
-// Interfaces should be part of the AST, see #2673
 let private isIgnoredLeafEntity (ent: FSharpEntity) =
-    // ent.IsInterface
     ent.IsEnum
     || ent.IsMeasure
-    || ent.IsFSharpAbbreviation //&& com.Options.Language <> Rust
```

Comment:
@alfonsogarciacaro It's fine, but we may bring back abbreviations for Rust in the future, as they're supported by the language.

Reply:
@ncave Ah, sorry for that. I was just trying to clean up. But please feel free to bring it back. Actually TypeScript could also benefit from abbreviations because some types can get too verbose in APIs. The tricky part is all types are "de-abbreviated" when entering the Fable AST. We can undo that but we will have to fix all transformations that assume types are never aliases.